subworkflows/DATA_TO_MODEL.nf

```diff
@@ -31,7 +34,7 @@ workflow DATA_TO_MODEL {
     VALIDATE(DOWNLOAD_SUBMISSION.output, "ready", params.validation_script)
     UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE(VALIDATE.output.map { tuple(it[0], it[2]) })
     ANNOTATE_SUBMISSION_AFTER_VALIDATE(VALIDATE.output)
-    SCORE(VALIDATE.output, UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE.output, ANNOTATE_SUBMISSION_AFTER_VALIDATE.output, params.scoring_script)
+    SCORE(VALIDATE.output, SYNAPSE_STAGE.output, UPDATE_SUBMISSION_STATUS_AFTER_VALIDATE.output, "ready", params.scoring_script)
```

Comment:
Looks like we're not waiting for the annotation task to be done before starting this one anymore. I think it might be good to add the annotation "ready" output back, since the final annotation runs as soon as SCORE is done. 2 annotation tasks running at the same time could get dicey.

Reply:
Good catch, that was not my intention.